indico/modules/events/models/persons.py

```diff
@@ -150,6 +152,30 @@ def get_full_name(self, last_name_first=True, last_name_upper=True, abbrev_first
     def __repr__(self):
         return format_repr(self, 'id', _text=self.full_name)
 
+    @staticmethod
+    def _build_user_filter(user):
```

Comment:
`classmethod` because it doesn't make any sense to have it as a method in `User`. It's really `EventPerson`-specific.